### PR TITLE
Fix casing in app title

### PR DESCRIPTION
**Pull request by jancborchardt**
Well at least you didn’t write "NextCloud" right? :wink:

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <info>
 	<id>twofactor_nextcloud_notification</id>
-	<name>TwoFactor via nextcloud notification</name>
+	<name>Two Factor Authentication via Nextcloud notification</name>
```
Comment (ChristophWurst) [inline appinfo/info.xml]:
```suggestion
	<name>Two-Factor Authentication via Nextcloud notification</name>
```

Comment (jancborchardt) [reply appinfo/info.xml]: Heh good catch! Actually thought about that the other day but forgot again. ;)

@nickvergessen now needs your review. :)

**Comment by nickvergessen** [comment]
/me searches quote from jan for "Only first case is uppercase in app names"

**Comment by jancborchardt** [comment]
> /me searches quote from jan for "Only first case is uppercase in app names"

Hehe yes I know. ;) Usually that’s the case, but "Two-Factor Authentication" is kind of a term I guess? We can change that to only "T" capitalized too. But "Nextcloud" should for sure be first-char uppercase. :)

**Pull request merged**